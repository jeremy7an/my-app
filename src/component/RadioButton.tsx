import { Fragment, FunctionComponent } from "react";

declare interface IRadioButtonProps {
label: string;
value: string;
name: string;
}

export const RadioButton: FunctionComponent<IRadioButtonProps> = (props: IRadioButtonProps) =>{
  return (
    <Fragment>
      <div className="flex_row">
        <label>{props.label}</label>
        <input type="radio" value ={props.value} name={props.name} />     
        </div>
    </Fragment>
  )

}