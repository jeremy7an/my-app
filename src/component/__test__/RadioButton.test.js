import renderer from 'react-test-renderer';
import { RadioButton } from "../RadioButton";


it("radio button matches snapshot", () => {
  const tree = renderer.create(<RadioButton label="label" value="value" name="name" />).toJSON();
  expect(tree).toMatchSnapshot();
  
});