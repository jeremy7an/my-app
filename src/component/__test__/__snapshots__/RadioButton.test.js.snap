// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`radio button matches snapshot 1`] = `
<div
  className="flex_row"
>
  <label>
    label
  </label>
  <input
    name="name"
    type="radio"
    value="value"
  />
</div>
`;
